day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,38 @@
+class Frog
+  attr_reader :number
+  def initialize (number)
+    @number = number
+  end
+
+  def remaining(enum)
+    number.replace(enum)
+    if (enum) == 0
+      puts "no more"
+  end
+
+  def info
+    "#{number}"
+  end
+end
+end
+
```

Comment:
You've got a lot of `puts` statements you are leaning on here! I would steer away from an OOP solution here. It can work, but it can over complicate this one. It could be good practice to redo this one, potentially getting some user input to inform how many times a loop should be run. This way, you can just type out repeated verses once, and your program will be more dynamic.